app/helpers/application_helper.rb

```diff
@@ -79,10 +79,10 @@ def time_field(object_name, method, options = {})
   def flash_message(type, text, now = nil)
     if now
       flash.now[type] ||= []
-      flash.now[type] << text
+      flash.now[type] << text.dup
```

Comment:
Why?

Reply:
I was trying to debug a frozen string literal error in the end I dont even think it was this, good catch, ill remove this.